server/app/views/applicant/NorthStarApplicantCommonIntakeUpsellView.java

```diff
@@ -0,0 +1,92 @@
+package views.applicant;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import controllers.AssetsFinder;
+import controllers.LanguageUtils;
+import controllers.applicant.ApplicantRoutes;
+import java.util.Locale;
+import modules.ThymeleafModule;
+import org.thymeleaf.TemplateEngine;
+import services.DeploymentType;
+import services.settings.SettingsManifest;
+
+public class NorthStarApplicantCommonIntakeUpsellView extends NorthStarApplicantBaseView {
+
+  @Inject
+  NorthStarApplicantCommonIntakeUpsellView(
+      TemplateEngine templateEngine,
+      ThymeleafModule.PlayThymeleafContextFactory playThymeleafContextFactory,
+      AssetsFinder assetsFinder,
+      ApplicantRoutes applicantRoutes,
+      SettingsManifest settingsManifest,
+      LanguageUtils languageUtils,
+      DeploymentType deploymentType) {
+    super(
+        templateEngine,
+        playThymeleafContextFactory,
+        assetsFinder,
+        applicantRoutes,
+        settingsManifest,
+        languageUtils,
+        deploymentType);
+  }
+
+  public String render(UpsellParams params) {
+    ThymeleafModule.PlayThymeleafContext context =
+        createThymeleafContext(
+            params.request(),
+            params.applicantId(),
+            params.profile(),
+            params.applicantPersonalInfo(),
+            params.messages());
+
+    String linkHref = settingsManifest.getCommonIntakeMoreResourcesLinkHref(params.request()).get();
+    String linkText = settingsManifest.getCommonIntakeMoreResourcesLinkText(params.request()).get();
+    String linkHtml =
+        "<a href=\""
+            + linkHref
+            + "\" target=\"_blank\" style=\"color: blue; text-decoration: underline;\">"
```

Comment:
you shouldn't need that style. It should be the default for links

Reply:
This is what it looks like without the style

<img width="1174" alt="Screenshot 2024-06-06 at 4 22 32 PM" src="https://github.com/civiform/civiform/assets/23390490/2fd5ce3e-e319-42a9-876a-74cce6e9e3a3">